src/OpenTelemetry.AutoInstrumentation.Native/logger_impl.h

```diff
@@ -97,11 +104,11 @@ template <typename TLoggerPolicy>
 LoggerImpl<TLoggerPolicy>::LoggerImpl()
 {
     spdlog::set_error_handler([](const std::string& msg) {
-        // By writing into the stderr was changing the behavior in a CI scenario.
-        // There's not a good way to report errors when trying to create the log file.
-        // But we never should be changing the normal behavior of an app.
-        // std::cerr << "LoggerImpl Handler: " << msg << std::endl;
-    });
+            // By writing into the stderr was changing the behavior in a CI scenario.
+            // There's not a good way to report errors when trying to create the log file.
+            // But we never should be changing the normal behavior of an app.
+            // std::cerr << "LoggerImpl Handler: " << msg << std::endl;
+        });
```

Comment:
nit: is this whitespace intentional?

Reply:
removed, seems unintentional. https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3558/commits/dc5caf6b437d523d8ed87c81ff3ca3395d2860c3